services/movie.py

```diff
@@ -25,16 +32,17 @@ def get_movie_by_id(movie_id: int) -> Movie:
 def create_movie(
     movie_title: str,
     movie_description: str,
-    genres_ids: list = None,
-    actors_ids: list = None,
+    genres_ids: Optional = None,
```

Comment:
Optional[list] not just optional
the same with actors_ids

Reply:
+ in function update_movie_session
Optional takes type also
Optional[int] it mean int or None